thirdweb/modules/nft.py

```diff
@@ -12,28 +12,42 @@
 
 
 class NftModule(BaseModule):
-    """ 
+    """
     NFT Methods
     """
 
     address: str
     __abi_module: NFT
 
+
     def __init__(self, address: str, client: Web3):
+        """
+        Initializing the class attributes
+        """
         super().__init__()
         self.address = address
+        """
+        NFT module contract address [https://thirdweb.com/: Dashboard: Project ➝ NFT Module]
+        """
+
         self.__abi_module = NFT(client, address)
+        """
+        The ABI makes calls the EVM. Client is default 'Web3' and Address is nft module contract address.
+        """
 
     def mint(self, arg: MintArg) -> NftType:
         """
-        Mints a new token
+        - Mints a new token. 
+        - Arguments passed: Note, a class is used -> MintArg(name, description, image_uri, properties) *Preferrably, using a link
```

Comment:
can we remove ` *Preferrably, using a link`?

Reply:
should probably be changed to the other format like in [this one](https://github.com/nftlabs/nftlabs-sdk-python/pull/15/files#diff-85ff59513ad491407f1f7bcaad29cf1d428e525190145629436e39407893033dR222) or the other way round. I like the second one more